distribution/src/bin-filemode-755/hz-start

```diff
@@ -34,9 +34,17 @@ fi
 
 . "$SCRIPT_DIR"/common.sh
 
-# HAZELCAST_CONFIG holds path to the configuration file. The path is relative to the Hazelcast installation (HAZELCAST_HOME).
+# HAZELCAST_CONFIG holds the configuration path.
+# If the the path is relative, it is relative to the Hazelcast installation directory (HAZELCAST_HOME).
+# The path can be absolute.
+# By default, it is set to $HAZELCAST_HOME/config/hazelcast.xml
 if [ -z "$HAZELCAST_CONFIG" ]; then
-    HAZELCAST_CONFIG="config/hazelcast.xml"
+    HAZELCAST_CONFIG="$HAZELCAST_HOME/config/hazelcast.xml"
```

Comment:
You don't have to duplicate here how the `HAZELCAST_CONFIG` is resolved. `HAZELCAST_HOME` will be added by the condition below :)

Reply:
Right, fixed at: https://github.com/hazelcast/hazelcast/pull/19908/commits/8e5e8f0ded022b492af64b46b7b8bddd988f6338